core/src/main/java/com/uber/m3/tally/sanitizers/Sanitize.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package com.uber.m3.tally.sanitizers;
+
+/**
+ * Sanitize returns a sanitized version of the input string value.
+ */
+public interface Sanitize {
```

Comment:
Please try for a different name. `Sanitize` next to `Sanitizer` confused me thoroughly

Reply:
The `Sanitizer` interface matches the `uber-go/tally` [Sanitizer](https://github.com/uber-go/tally/blob/master/sanitize.go#L77) interface.

The `Sanitize` interface is meant to match the `uber-go/tally` [SanitizeFn](https://github.com/uber-go/tally/blob/master/sanitize.go#L57) interface.  Name it `SanitizeFn` too?

Reply:
Please elaborate on what this interface is needed for